pkg/kubelet/kubelet_pods.go

```diff
@@ -1512,11 +1512,15 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 		statuses[container.Name] = status
 	}
 
+	// Copy the slice before sorting it
```

Comment:
Suggest replacing this with "Clone the slice to make sorting thread-safe"

Reply:
Done ! 👍